### PR TITLE
Shifted speaker test button to audiobait page.

### PR DESCRIPTION
**Pull request by Davo36**
## API version update?
- If the API has been changed please update the API version

^^ Dunno about that.

This handles part 1 of #90.

### CODE REVIEW

**Comment by Davo36** [comment]
Menno, I have been working on this.  The schedule sounds can now be played as well.

**Comment by gferraro** [comment]
Could add a button to play again? in the modal popup

File: html/audiobait.html [hunk 1]
```diff
@@ -231,7 +278,53 @@ <h4 class="modal-title">Recent Log Entries</h4>
...
+            modal.find('button#crossCloseButton').prop("disabled", true);
+
+            fetch(
+              "/audiobait-test-sound/" + fileName +  "/" + volume,
+              {method: "GET"}
+
+            ).then(function(response) {
+                // Show a symbol indicating whether the test was successfull or not.
+                if (response.ok) {
+                    modal.find('h4#statusIndicator').html("<h4>&#x2714;</h4>");// Check mark
```
[the remark this answers is not in the document]
Comment (Davo36) [reply html/audiobait.html]: I did simplify this a bit.  But still uses the html property.  The check mark I get using the text property like this: `modal.find('h4#statusIndicator').text("\u2713 Available"); ` is tiny, the html version seems to stand out a lot more for some reason.

**Comment by Davo36** [comment]
Done.

**Pull request merged**